pytket/binders/circuit/Circuit/main.cpp

```diff
@@ -119,8 +121,8 @@ void def_circuit(py::class_<Circuit, std::shared_ptr<Circuit>> &pyCircuit) {
           py::arg("n_qubits"), py::arg("name") = std::nullopt)
       .def(
           py::init<unsigned, unsigned, std::optional<std::string>>(),
-          "Constructs a circuit with a given number of quantum and "
-          "classical bits\n\n:param n_qubits: The number of qubits in "
```

Comment:
Most of these changes are autoformatting? Not sure why the diff is so big with only a minor edit.